type_stubs/README.md

```diff
@@ -0,0 +1,12 @@
+# Type stubs
+
+[mypy]: https://mypy.readthedocs.io
+[type hint annotations]: https://docs.python.org/3/glossary.html#term-type-hint
+[type stub files]: https://docs.python.org/3/library/typing.html#stub-files
```

Comment:
The "type stub files" link isn't working for me. Maybe https://mypy.readthedocs.io/en/stable/stubs.html instead?

Reply:
Fixed!